src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -1,77 +1,92 @@
 package wooteco.subway.line.dao;
 
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.sql.DataSource;
 import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
 import org.springframework.jdbc.core.simple.SimpleJdbcInsert;
 import org.springframework.stereotype.Repository;
 import wooteco.subway.line.domain.Line;
 import wooteco.subway.line.domain.Section;
 import wooteco.subway.line.domain.Sections;
 import wooteco.subway.station.domain.Station;
 
-import javax.sql.DataSource;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
-
 @Repository
 public class LineDao {
+
+    private static RowMapper<Line> ROW_MAPPER() {
```

Comment:
```suggestion
    private static RowMapper<Line> rowMpper() {
```

메서드는 카멜케이스로 작성하는 것이 관례입니다 :)